src/components/DashboardFilters/DashboardFilters-container.tsx

```diff
@@ -0,0 +1,172 @@
+
+  const checkActiveBlockchains = (label:string) => {
+    const activeBlockchains = urlParams.blockchains.split(',');
+    return activeBlockchains.indexOf(label) !== -1;
+  }
+
+  const checkActiveZoom = (label: string) => {
+    const zoom = urlParams.to - urlParams.from;
+    return (label === '1 day' && zoom === 86400) ? true : (label === '7 days' && zoom !== 86400) ? true : false;
+  };
+
+  const checkActiveTopList = (label: string) => label === urlParams.limit ? true : false;
+
+  const [activeBlockchainButtons, setBlockchainButtons]: [Record<string, boolean>, Function] = useState({
+    'BTC': checkActiveBlockchains('BTC'),
+    'ETH': checkActiveBlockchains('ETH'),
+    'XRP': checkActiveBlockchains('XRP'),
+    'LTC': checkActiveBlockchains('LTC'),
+    'BCH': checkActiveBlockchains('BCH'),
+    'XTZ': checkActiveBlockchains('XTZ'),
+    'ADA': checkActiveBlockchains('ADA'),
+    'EOS': checkActiveBlockchains('EOS'),
+    'XLM': checkActiveBlockchains('XLM'),
+  });
+
+
+  const [activeZoomButtons, setZoomButtons]: [Record<string, boolean>, Function] = useState({
+    '1 day': checkActiveZoom('1 day'),
+    '7 days': checkActiveZoom('7 days'),
+    // '1 month': false,
+    // '3 months': false,
+    // '1 year': false,
+    // 'All': false
+  });
+
+  const [activeTopListButtons, setTopListButtons]: [Record<string, boolean>, Function] = useState({
+    '10': checkActiveTopList('10'),
+    '100': checkActiveTopList('100'),
+    '1000': checkActiveTopList('1000')
+  });
+
+  const [filters, setFilters]:[FiltersProps, Function] = useState({});
+
+  let setURL: boolean= false;
+  let newFilters: FiltersProps = {};
+
+  const classes = useFiltersStyles();
+
+  const blockchainFilterHandler = (buttonLabel: string) => {
+    if (buttonLabel === 'ETH' || buttonLabel === 'XTZ') {
+      const buttons = { ...activeBlockchainButtons };
+      buttons[buttonLabel] = !buttons[buttonLabel];
+      setBlockchainButtons({ ...buttons });
+    }
+  };
+
+  const zoomFilterHandler = (buttonLabel: string) => {
+    const buttons = { ...activeZoomButtons };
+    Object.keys(buttons).filter((label: string) => (buttons[label] = false));
+    buttons[buttonLabel] = true;
+    setZoomButtons({ ...buttons });
+  };
+
+  const zoomTopListHandler = (buttonLabel: string) => {
+    const buttons = { ...activeTopListButtons };
+    Object.keys(buttons).map((label: string) => (buttons[label] = false));
+    buttons[buttonLabel] = true;
+    setTopListButtons({ ...buttons });
+  };
+
+  const filterHandler = (buttonLabel: string, filterType: Record<string, boolean>) => {
+    filterType === activeBlockchainButtons && blockchainFilterHandler(buttonLabel);
+    filterType === activeZoomButtons && zoomFilterHandler(buttonLabel);
+    filterType === activeTopListButtons && zoomTopListHandler(buttonLabel);
+  };
+
+  const activeFilters = (filterObj: Record<string, boolean>) => Object.keys(filterObj).filter((item: string) =>  filterObj[item] === true);
```

Comment:
Można jak wyżej albo:
1) !!filterObj[item]
2) Boolean(filterObj[item])
To tylko tak FYI ;)